### PR TITLE
Added: Cart Page

### PR DESCRIPTION
**Pull request by pratik-kumar-sahu**
- Added Cart Page
- Live link --> https://deploy-preview-5--shop-up.netlify.app/pages/cart.html

### CODE REVIEW

**Review by Aazam10 — APPROVED** [review]
Good work.

File: styles/ShopUp.css [hunk 1]
```diff
@@ -125,6 +144,33 @@
     width: 25rem;
     height: auto;
     overflow: hidden;
+    position: relative;
+}
+
+.product-icon {
+    font-size: 1.8rem;
+    padding: 0.6rem;
```
Comment (Sharmaryan) [inline styles/ShopUp.css]: you can use px for values less than 10px

**Review by Sharmaryan — APPROVED** [review]
suggested a change

File: styles/ShopUp.css (hunk 1 above)
Comment (pratik-kumar-sahu) [reply styles/ShopUp.css]: Sure I'll look into that Thanks for reviewing

**Pull request merged**